index.js

```diff
@@ -0,0 +1,60 @@
+var fs = require('fs-extra'),
+    path = require('path'),
+    cssstats = require('css-statistics'),
+    json2html = require('json2html');
+
+module.exports = CssReportGeneratorPlugin;
+
+function CssReportGeneratorPlugin(options) {
+    this.options = options;
+}
+
+CssReportGeneratorPlugin.prototype.apply = function(compiler) {
+    var that = this;
+    if (!this.options.disabled) {
+        compiler.plugin('done', function() {
+            that.generateRaports();
+            console.log("Generated CSS raports!");
```

Comment:
starajmy się w kodzie plugina nie używać console.log'ów

jeśli koniecznie chcemy je mieć, to warto dodać flagę `verbose`, która domyślnie będzie ustawiona na false.

Dzięki temu np.:
- console.log nie pokaże się w testach jednostkowych,
- udostępnimy możliwość włączenia log'ów podczas debugowania